rustler/src/types/map.rs

```diff
@@ -25,8 +25,8 @@ impl<'a> Term<'a> {
     /// ```elixir
     /// List.zip(keys, values) |> Map.new()
     /// ```
-    #[cfg(nif_2_14)]
-    pub fn map_from_arrays(env: Env<'a>, keys: &[Term], values: &[Term]) -> NifResult<Term<'a>> {
+    #[cfg(nif_version_2_14)]
+    pub fn map_from_arrays(env: Env<'a>, keys: &[Term<'a>], values: &[Term<'a>]) -> NifResult<Term<'a>> {
```

Comment:
When you fixed this to `nif_version_2_14` did it fail to compile without the lifetimes on `Term`? I suspect it should have, just curious.

Reply:
It did not fail. I don't know what exactly is being tested by the test but everything passed and the debug I had in that version was called.

Reply:
Cool thanks!